wikipedia-dark.user.css

```diff
@@ -2604,8 +2604,9 @@ regexp("https?:\/\/wiki\.(rpcs3\.net|(archlinux|mozilla)\.(org|jp))\/.*$") {
   .mw-rcfilters-ui-filterMenuOptionWidget.oo-ui-flaggedElement-muted:not(.oo-ui-optionWidget-selected) {
     background-color: var(--gray-2);
   }
-  div[style*="background-color:#f9f9f9"] {
+  div[style*="background-color:#f9f9f9"], div[style*="background-color:#fffaf5"] {
     background-color: var(--gray-2) !important;
+    border-color: rgb(85, 85, 85) !important;
```

Comment:
I think this could fix it, but not sure as the message isn't showing up for me any more.

Reply:
2 things.

1) Dont add borders to rules that dont have a border. In this case looks for a existing rule that matches background `var(--gray-2)` and has a border-color of `var(--gray-4)` or `var(--gray-5)`
2) Use the existing variables for colors. In this case it would be`var--gray-4) or var(--gray-5)`

Also, cant really see this either, Ive cleared my cookies and still cant see this message.